algoflex_frontend/src/services/WebSocket.client.ts

```diff
@@ -0,0 +1 @@
+export const webSocketLink = "ws://staging-algoflex.herokuapp.com";
```

Comment:
Ça marche sans le wss ?
```suggestion
export const webSocketLink = "wss://staging-algoflex.herokuapp.com";
```